benchmarks/tasks.py

```diff
@@ -39,32 +78,33 @@ def set_testjob_results(self, testjob):
     if not testjob.initialized:
         testjob.testrunnerclass = tester.get_result_class_name(testjob.id)
         testjob.initialized = True
-        testjob.save()
         tester = getattr(testminer, testjob.testrunnerclass)(
             testjob.testrunnerurl, username, password
         )
 
     if testjob.status not in ["Complete", "Incomplete", "Canceled"]:
-        logger.debug("Saving job({0}) status: {1}".format(testjob.id, testjob.status))
-        testjob.save()
+        logger.debug("Job({0}) status: {1}".format(testjob.id, testjob.status))
         return
```

Comment:
Will it return 'None'/'False'? There is a condition checking what is returned from this function